src/App.tsx

```diff
@@ -21,14 +24,21 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              { todos.length ? (<TodoList />
+              ) : (
+                query === '' && <Loader />
```

Comment:
Need change. If the query is empty I see the loader, but I need to see an empty result.